backend/src/infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -1,11 +1,10 @@
-using Microsoft.EntityFrameworkCore;
-
-namespace infrastructure.Data;
-
-public class ApplicationDbContext : DbContext
-{
-    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
-    {
-    }
-    public DbSet<TodoListModel> TodoLists { get; set; }
-}
+using Microsoft.EntityFrameworkCore;
+
+namespace infrastructure.Data;
+
+public class ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : DbContext(options)
+{
+    public required DbSet<TodoListModel> TodoLists { get; set; }
+    public required DbSet<TodoItemModel> Todos { get; set; }

```

Comment:
I think it is better to include the whole name of the model: `TodoItems`

Reply:
I did thins change, but I have renamed the tables in db to all use the same naming schema